ui/packages/app/src/components/Dataset/ExampleComponents/Dataframe.svelte

```diff
@@ -43,6 +43,10 @@
 				} catch (e) {
 					console.error(e);
 				}
+			})
```

Comment:
I think we need to make this modification to the `Dataset` component so that the image urls display when launching a private space that's loaded

```ts
	let samples_dir: string = root_url ? "proxy=" + root_url + "file=" : root + "file="
```

### Before
![image](https://user-images.githubusercontent.com/41651716/217097813-2d28d38b-6c02-4a60-992f-508c7ccab7c7.png)

### After
![image](https://user-images.githubusercontent.com/41651716/217097980-27e405bd-fa28-4f2b-97a2-d75d74f4c886.png)

Reply:
Interesting good catch